packages/reqresp/test/fixtures/messages.ts

```diff
@@ -164,8 +164,8 @@ export const sszSnappySignedBeaconBlockAltair: MessageFixture = {
     "0x003f0000ee14ab0df8031064000000dafe01007a01000c995f0100010100090105ee70000d700054ee44000d44fe0100fecc0011cc0c400100003e0400fe01008e0100",
   ].map((d) => fromHexString(d)),
   chunks: [
-    "0xb404",
-    "0xff060000734e6150705900420000bab7f8feb4041064000000dafe01007a01000c995f0100010100090105ee70000d700054ee44000d44fe0100fecc0011cc0c7c0100003e0400fe0100fe01007e0100",
+    "0xf803",
+    "0xff060000734e61507059003f0000ee14ab0df8031064000000dafe01007a01000c995f0100010100090105ee70000d700054ee44000d44fe0100fecc0011cc0c400100003e0400fe01008e0100",
```

Comment:
Why did these values change?  Seems unrelated to changing the runner

Reply:
it's the correct value, was not checked before because of wrong preset value.